src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,9 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) =>
+  comments.map(comment => (
+    <div className="CommentList" key={comment.id}>
```

Comment:
Now you create div with className="CommentList" for each comment. Looks not correct

Just use this div as a wrapper and do mapping inside. Pass key directly to the component